### PR TITLE
Localization Preparation SLE Micro

### PR DESCRIPTION
**Pull request by taroth21**
### Description

Prepare localization drop. Added a DEC for html build and a DEF file.

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ----------
| [ x] 15 SP3  | *(no backport necessary)*
| [ ] 15 SP2  | [ ]
| [ ] 15 SP1  | [ ]
| [ ] 15 SP0  | [ ]

### CODE REVIEW

File: DEF-SLE-Micro-manuals [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# HTML must come first and covers the complete set in HTML output format
+html       SLE-Micro-html             common_copyright_gfdl.xml ok https://api.suse.de SUSE:SLE-15-SP3:GA
```
Comment (fsundermeyer) [inline DEF-SLE-Micro-manuals:2]:
```suggestion
html       SLE-Micro-all             common_copyright_gfdl.xml ok https://api.suse.de SUSE:SLE-15-SP3:GA
```

File: DEF-SLE-Micro-manuals [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+pdfsub     DC-SLE-Micro-autoyast      common_copyright_gfdl.xml
```
Comment (fsundermeyer) [inline DEF-SLE-Micro-manuals:3]:
```suggestion
pdfsub     SLE-Micro-autoyast      common_copyright_gfdl.xml
```

File: DEF-SLE-Micro-manuals [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+pdfsub     DC-SLE-Micro-installation  common_copyright_gfdl.xml
```
Comment (fsundermeyer) [inline DEF-SLE-Micro-manuals:4]:
```suggestion
pdfsub     SLE-Micro-installation  common_copyright_gfdl.xml
```

**Review by fsundermeyer — CHANGES_REQUESTED** [review]
The DC-SLE-Micro-html is not needed--it is identical to the already existing DC-SLE-Micro-all.
The DC-file entries in the DEF file need to be listed without "DC-"

**Comment by taroth21** [comment]
Thanks for the review, Frank. I incorporated your proposals, we should be good to merge once Travis has finished.

**Pull request merged**